src/groups_v2/model.rs

```diff
@@ -105,31 +133,31 @@ pub enum GroupChange {
         #[derivative(Debug = "ignore")]
         profile_key: ProfileKey,
     },
-    // for open groups
     NewPendingMember(PendingMember),
     DeletePendingMember(Uuid),
     PromotePendingMember {
         uuid: Uuid,
         #[derivative(Debug = "ignore")]
         profile_key: ProfileKey,
     },
-    // when admin control is enabled
-    NewRequestingMember(RequestingMember),
-    DeleteRequestingMember(Uuid),
-    PromoteRequestingMember {
-        uuid: Uuid,
-        role: Role,
-    },
-    // group metadata
     Title(String),
     Avatar(String),
     Timer(Option<Timer>),
-    Description(Option<String>),
     AttributeAccess(AccessRequired),
     MemberAccess(AccessRequired),
     InviteLinkAccess(AccessRequired),
+    NewRequestingMember(RequestingMember),
+    DeleteRequestingMember(Uuid),
+    PromoteRequestingMember {
+        uuid: Uuid,
```

Comment:
Same question

Reply:
This is Aci, by SA sources. This makes sense, because if I request a group membership, I'll start the process directly with Aci, w.r.t. if someone is being invited, it may or may not be a known recipient.